dbhydra/dbhydra_core.py

```diff
@@ -1772,17 +1786,15 @@ def dict_to_df(dictionary, column1, column2):
     df = pd.DataFrame(list(dictionary.items()), columns=[column1, column2])
     return (df)
 
-from pydantic import BaseModel
-from typing import List, Dict
 
 class AbstractModel(abc.ABC, BaseModel):
```

Comment:
check

Reply:
Imports shifted to the top as a part of a small clean up.